frontend/pages/admin/IntegrationsPage/cards/Certificates/helpers.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  ICustomScepProxy,
+  IDigicertCertificate,
+  IScepIntegration,
+} from "interfaces/integration";
+
+export interface ICertAuthority {
+  id: string;
+  name: string;
+  certificateAuthority: string;
+}
+
+export const generateListData = (
+  ncspProxy?: IScepIntegration | null,
```

Comment:
What does `ncspProxy` stand for? Should this be `ndesProxy`?

Reply:
good catch. think this was just typo. updated.